maintainers/maintainer-list.nix

```diff
@@ -1646,6 +1646,12 @@
     githubId = 1382175;
     name = "Oliver Matthews";
   };
+  cyplo = {
```

Comment:
I'd like to humbly request that you split this out to a separate commit, please: `maintainers: add cyplo`

Also, you should be below `cypherpunk2140` -- `h` comes before `l` ;)

Reply:
must have been late :D